### ISSUE TITLE
Action tag doesn't work when the field is embedded.

### ISSUE THREAD

**Issue by grezniczek** [jangari/redcap_hidesubmit#10]
@jangari , I just noticed that when a (descriptive) field that has the `@HIDESUBMIT` action tag is embedded, the action tag fails to work.

It would be nice if it continued to work. There are, of course, workarounds.

**Comment by jangari** [comment]
Also reported [here](https://redcap.vumc.org/community/post.php?id=264484).

@grezniczek what did you have in mind regarding workarounds?

**Comment by grezniczek** [comment]
@jangari - Long time ago. No idea what I had in mind then. But I guess that making a "duplicate" field with the `@HIDESUBMIT` field that is not embedded (and potentially making it invisible in other ways using CSS) is a workaround.

As for a coded solution ... you'd have to check field embeddings for candidate fields as well:

<img width="509" height="133" alt="Image" src="https://github.com/user-attachments/assets/45bbfb01-8089-43c7-a4dc-a6e5d320ed00" />

Something like `$('.rc-field-embed[var="desc"]').is(':visible')`.

**Comment by jangari** [comment]
Yep, I figured you meant something along the lines of dealing with the limitation. 

Thanks for that pointer. I should be able to push a fix soon. Sorry it to two years to get to!
